### ISSUE TITLE
indentation rule requires multi-line values with variables to be indented more than the previous line

### ISSUE THREAD

**Issue by FiniteLooper** [sasstools/sass-lint#1191]
Related to #1190, I've found that using multi-line property values with variables seems to require each line be indented more than the last one from the `indentation` rule!

I am running Sass Lint version 1.12.1 from the command line, and from the VSCode IDE extension.

Given this code, I lint violations on the last 3 lines - I've written the output in the comment on each line
```scss
@keyframes shine {
  to {
    background-position:
    100% 0,                                                 //No violation
    $skeleton-content-left ($skeleton-content-spacing * 1), //No violation
    $skeleton-content-left ($skeleton-content-spacing * 2), //Expected indentation of 6 spaces but found 4.
    $skeleton-content-left ($skeleton-content-spacing * 3), //Expected indentation of 8 spaces but found 4.
    $skeleton-content-left ($skeleton-content-spacing * 4); //Expected indentation of 10 spaces but found 4.
  }
}
```

So apparently it expects my code to be written like this, which seems completely ridiculous.  The following code produces no violations from the `indentation` rule.

```scss
@keyframes shine {
  to {
    background-position:
    100% 0,
    $skeleton-content-left ($skeleton-content-spacing * 1),
      $skeleton-content-left ($skeleton-content-spacing * 2),
        $skeleton-content-left ($skeleton-content-spacing * 3),
          $skeleton-content-left ($skeleton-content-spacing * 4);
  }
}
```

**Comment by ChrisMBarr** [comment]
I assume this has been dealt with by this point - it's been several years now so I'm closing this issue to get it out of my list
